### PR TITLE
Fix CONTRIBUTING.md link

### PR DESCRIPTION
**Pull request by shuheiktgw**
The link is expired and I guess this is the right entry point for new contributors?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Contribution guidelines
 
 So you want to hack on Knative `pkg`? Yay! Please refer to Knative's overall
-[contribution guidelines](https://github.com/knative/docs/blob/master/community/CONTRIBUTING.md)
+[contribution guidelines](https://github.com/knative/community/blob/master/README.md)
```
Comment (vagababov) [inline CONTRIBUTING.md:4]: https://knative.dev/contributing/ looks like a proper link

**Review by vagababov — COMMENTED** [review]
/ok-to-test

**Comment by shuheiktgw** [comment]
@googlebot I signed it!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Contribution guidelines
 
 So you want to hack on Knative `pkg`? Yay! Please refer to Knative's overall
-[contribution guidelines](https://github.com/knative/docs/blob/master/community/CONTRIBUTING.md)
+[contribution guidelines](https://knative.dev/contributing/)
 to find out how you can help.
```
Comment (mattmoor) [inline CONTRIBUTING.md:5]: Format markdown:
```suggestion
[contribution guidelines](https://knative.dev/contributing/) to find out how you
can help.
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github | grep -v docs/cmd/)`

File: CONTRIBUTING.md (hunk 1 above)
Comment (shuheiktgw) [reply CONTRIBUTING.md]: Thank you, fixed it!

**Review by mattmoor — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**